iFixit/react-commerce
# Strapi: Deploy with actions

We previously used cimpler to deploy strapi. We are trying to remove all
instances of cimpler, and this is the last one. This creates a copy of the
strapi-deploy script that cimpler uses. It's slightly modified because some
of the checks it did can be more easily expressed in the workflow file.

CR:
--
I think we eventually will want to transform the bash script into actual
workflow steps, but for now a copy of the existing/working bash script
should be enough.

QA:
--
Idk how we want to test this. Might have to just test this on deploy.

On Deploy:
--
Deactivate the webhook that sends the push event to ubreakit. Deploy
and verify that all went well.

Closes: https://github.com/iFixit/react-commerce/issues/2175

qa_req 0
QA on deployment
